charts/zipkin/ci/helmfile.yaml

```diff
@@ -0,0 +1,55 @@
+---
+# install via `helmfile -f charts/zipkin/ci/helmfile.yaml sync`
+repositories:
+  - name: prometheus-community
+    url: https://prometheus-community.github.io/helm-charts
+
+# Prometheus is too much to configure manually in a test yaml. We need the CRD
```

Comment:
`in a test yaml so we use helm.`

Maybe, I wasn't quite sure what the intention is of this comment, made a guess

Reply:
good guess